packages/terra-application-navigation/src/common/_PopupMenu.jsx

```diff
@@ -165,6 +173,7 @@ const PopupMenu = ({
   if (onSelectFooterItem) {
     endContent = (
       <Button
+        id={logoutId || undefined}
```

Comment:
```suggestion
        id={logoutId}
```
i feel both are same.